test/unit/client.test.js

```diff
@@ -285,9 +286,10 @@ describe('Client', function() {
       setTimeout(function() { return test.server.setup(); }, 10);
 
       var address = test.server.address();
-      test.client.policy.reconnect = {
-        retries: 5, strategy: 'fibonacci', forever: true
-      };
+      test.client.policy = test.client.policy = new Policy({
```

Comment:
`test.client.policy = test.client.policy = `

Man, that is _really_ well assigned.